plugins/catalog-backend-module-github/src/providers/GithubMultiOrgEntityProvider.ts

```diff
@@ -248,53 +248,49 @@ export class GithubMultiOrgEntityProvider implements EntityProvider {
       : await this.getAllOrgs(this.options.gitHubConfig);
 
     for (const org of orgsToProcess) {
-      try {
-        const { headers, type: tokenType } =
-          await this.options.githubCredentialsProvider.getCredentials({
-            url: `${this.options.githubUrl}/${org}`,
-          });
-        const client = graphql.defaults({
-          baseUrl: this.options.gitHubConfig.apiBaseUrl,
-          headers,
+      const { headers, type: tokenType } =
```

Comment:
Can we add a test around this change, please?

Reply:
Just to clarify, the test you refer to is to validate that the mutation does not in fact get triggered if a request fails, right?

Reply:
In any case such a test would make sense so I have added it

Reply:
Yes, thanks for adding this 👍